docs/guides/tanzu-mission-control_cluster.md

```diff
@@ -78,8 +78,14 @@ resource "tanzu-mission-control_cluster" "create_tkgs_workload" {
       topology {
         control_plane {
           class             = "<class>"        // Required
-          storage_class     = "<storage-class" // Required
+          storage_class     = "<storage-class>" // Required
           high_availability = false            // Default: false
+          volumes {
+            capacity          = volume-capacity
```

Comment:
`<volume-capacity>`

Reply:
Capacity is a float type variable, we cannot have it in the format `<volume-capacity>` or `"<volume-capacity>"`